integration_tests/mockApis/premises.ts

```diff
@@ -44,9 +44,25 @@ const stubPremisesSummary = (premises: ExtendedPremisesSummary) =>
     },
   })
 
+const stubSinglePremises = (premises: Premises) =>
```

Comment:
is it worth including a mention in this commit message as to why we need to fetch the premises information?

Reply:
have done